components/suggestion/channel_mention_provider.jsx

```diff
@@ -3,15 +3,13 @@
 
 import React from 'react';
 
-import {getMyChannels, getChannel, getMyChannelMemberships} from 'mattermost-redux/selectors/entities/channels';
+import {getMyChannels, getMyChannelMemberships} from 'mattermost-redux/selectors/entities/channels';
 
 import {sortChannelsByTypeAndDisplayName} from 'mattermost-redux/utils/channel_utils';
 
-import {ChannelTypes} from 'mattermost-redux/action_types';
-
 import store from 'stores/redux_store.jsx';
 
-import {Constants} from 'utils/constants';
+import {Constants} from 'utils/constants.jsx';
```

Comment:
Why this change is needed?

Reply:
I think that was automated linting.

Reply:
0/5 on this change now, but it's going to get reverted when that file is switched to TypeScript. You'll probably need to figure out how to disable that in the future

Reply:
That was `npm run fix` by the way.

Reply:
Anyway, I would revert this change because is going to be typescript soon (I think).

Reply:
No problem :+1: